src/main/webapp/app/shared/select/NcitCodeSelect.tsx

```diff
@@ -26,16 +26,17 @@ export const parseNcitUniqId = (ncitId: string) => {
   }
 };
 
+export function getOptionFromNcit(ncit: INciThesaurus) {
+  return {
```

Comment:
The following is the current drug option, we should use the same. We should also add ncit code in both places to support the feature.

```

          value: entity.id,
          label: entity.name,
          uuid: entity.uuid,
          synonyms: entity.nciThesaurus?.synonyms,
        }
```

Reply:
Good call for adding synonyms. This will allow the drug select to support synonyms for NCITs. However, I don't think it makes sense for an NCIT to have a uuid. Since that is only relevant with firebase, and NCITs are not in firebase until the user hits `Confirm`, I think it makes sense to generate the uuid at that point. This is fine, since there is np reason the `ModifyTherapyModal` would try to access an NCIT's uuid.